components/Projects/ProjectPage/index.tsx

```diff
@@ -34,59 +28,57 @@ const ProjectPage = () => {
                 Projects
               </BreadcrumbItem>
               <BreadcrumbItem active tag='span' className='active-tab'>
-                Ignite Mentoring
+                {data.name}
               </BreadcrumbItem>
             </Breadcrumb>
           </Col>
           <Col lg={9}>
             <div className='mb-5'>
-              <h1 className='display-4 m-0 mb-4 monospace'>Ignite Mentoring</h1>
+              <h1 className='display-4 m-0 mb-4 monospace'>{data.name}</h1>
 
               <Row className='align-items-center mb-4 d-lg-none monospace'>
                 <Col xs={6} className='d-flex align-items-center'>
-                  <i className='material-icons-sharp mr-3'>devices</i>
-                  Web Development
+                  <i className='material-icons-sharp mr-3'>{data.icon}</i>
+                  {data.type}
                 </Col>
                 <Col xs={6} className='d-flex align-items-center'>
                   <i className='material-icons-sharp mr-3'>date_range</i>
-                  February 2019
+                  {data.date}
                 </Col>
               </Row>
 
               <p>
-                Lorem ipsum dolor sit amet consectetur, adipisicing elit.
-                Possimus tempore quia deserunt praesentium maiores optio
-                blanditiis voluptas labore repellat, excepturi, quidem aliquid
-                eum soluta unde ipsum repellendus molestias consectetur
-                eligendi! Lorem ipsum dolor sit, amet consectetur adipisicing
-                elit. Eius, reiciendis a adipisci sequi placeat porro eum
-                laborum ipsum voluptatum excepturi aliquid cum, commodi rem ad
-                repudiandae! Possimus, facilis minima. Tempore!
+                {data.desc}
               </p>
             </div>
             <div className='mb-5'>
               <h3 className='font-weight-black mb-4'>Technologies used</h3>
-              <TechList />
+              <TechList data={data.tech} />
             </div>
             <div>
               <h4 className='font-weight-black mb-4'>Members</h4>
+              {data.members.map((member: string, i: number) => (
+                <p key={i}>{member}</p>
+              ))}
             </div>
           </Col>
           <Col lg={3}>
             <div className='d-none d-lg-block mb-5 monospace'>
               <div className='d-flex align-items-center py-3'>
                 <i className='material-icons-sharp mr-3'>devices</i>
-                Web Development
+                {data.type}
               </div>
               <div className='d-flex align-items-center py-3'>
                 <i className='material-icons-sharp mr-3'>date_range</i>
-                February 2019
+                {data.date}
               </div>
               <Button
```

Comment:
This will have to be an `a` tag modified to look like a button.

Reply:
Wat

Reply:
since the button links to another website, semantically, it needs to be an anchor tag, but styled to look like a button for the normal user